package.json

```diff
@@ -92,7 +93,7 @@
     "vite-plugin-environment": "^1.1.3",
     "vitest": "0.33.0",
     "vue-class-component": "7.2.6",
-    "vue-plugin-helper-decorator": "^0.0.12",
+    "vue-plugin-helper-decorator": "0.0.11",
```

Comment:
What issue does this pre-empt/solve?

Reply:
I thought it solved the above issue, but it didn't 😟

It was fixed at first but after shutting down the server and running again, I got the exact same issue. I'll probably revert this back.

Reply:
I feel for ya. Good job looking into it.

Reply:
Obsolete package. Now using "vue-plugin-helper-decorator". Resolving.